docs/quickstart.rst

```diff
@@ -0,0 +1,83 @@
+.. toctree::
+   :maxdepth: 1
+
+Quickstart
+==========
+
+Installation
+------------
+
+.. code-block:: bash
+
+    pip install rikai[torch]
+
+Use `Rikai` in the machine learning life cycle
+----------------------------------------------
+
+Lets get started from the feature engineering in `Spark`_.
+
+
+.. code-block:: python
+
+    from pyspark.ml.linalg import DenseMetrix
```

Comment:
clarify that you need pyspark setup (and point to pyspark setup instructions otherwise) ?

Reply:
How does databricks deploy these jars? should we use `spark.jar.packages` or do we expect a deployment method that install rikai by default on cluster @changhiskhan